themes/finna2/js/finna-record.js

```diff
@@ -150,16 +150,26 @@ finna.record = (function finnaRecord() {
   }
 
   function initHoldingsControls() {
+    $('.record-holdings-table:not(.electronic-holdings) .holdings-container-heading').on('keydown', function onClickHeading(e) {
+      if (e.keyCode === 13 || e.keyCode === 32) {
+        e.preventDefault();
+        $('.record-holdings-table:not(.electronic-holdings) .holdings-container-heading').click();
+      }
+    });
     $('.record-holdings-table:not(.electronic-holdings) .holdings-container-heading').on('click', function onClickHeading(e) {
+      if ($('.location', this).parent().attr('aria-expanded') === 'false') {
```

Comment:
Eikö `$('.location', this).parent()` ole käytännössä sama kuin `$(this)`? Eli turha etsiä mitään, kun voisi tehdä vaikka näin:
`$(this).attr('aria-expanded', function (i, attr) { return attr === 'true' ? 'false' : 'true'; });`